code42/py42
# fix logging regression

This fixes a regression that caused `import py42.settings.debug` to fail ("`from py42.settings import debug` still worked).

The problem was caused by the fact that `import` statements can only import modules and not variables or classes, which require using `from module import symbol`.

The solution essentially fools python into thinking that our new class is a module.

To test:
- Install the latest version of the cli into a virtual env.
- Observe that simply typing `code42` results in error re: `import py42.settings.debug` not working
`pip install -e .[dev]` this branch into that same virtual env
- Observe that typing `code42` now behaves as normal

All existing tests re: debug settings still passing.
